packages/notifications/src/elements/Notification.tsx

```diff
@@ -15,30 +15,22 @@ import { validationIcons, validationHues } from '../utils/icons';
 /**
  * @extends HTMLAttributes<HTMLDivElement>
  */
-export const Notification = forwardRef<HTMLDivElement, INotificationProps>(
-  ({ role, ...props }, ref) => {
-    const Icon = props.type ? validationIcons[props.type] : InfoStrokeIcon;
-    const hue = props.type && validationHues[props.type];
+export const Notification = forwardRef<HTMLDivElement, INotificationProps>((props, ref) => {
+  const Icon = props.type ? validationIcons[props.type] : InfoStrokeIcon;
+  const hue = props.type && validationHues[props.type];
 
-    return (
-      <StyledNotification
-        ref={ref}
-        type={props.type}
-        isFloating
-        {...props}
-        role={role === undefined ? 'status' : role}
-      >
-        {props.type && (
-          <StyledIcon hue={hue}>
-            <Icon />
-          </StyledIcon>
-        )}
+  return (
+    <StyledNotification ref={ref} type={props.type} isFloating role="alert" {...props}>
```

Comment:
```suggestion
    <StyledNotification ref={ref} type={type} isFloating role="alert" {...props}>
```

Not exactly related to this PR, but should `type` be structured out in the parameters above so that it doesn't (re)spread?